server.go

```diff
@@ -42,9 +43,18 @@ func NewServer(c *cli.Context) (*server, error) {
 		}
 		surls = append(surls, surl)
 	}
+
+	t := http.DefaultTransport.(*http.Transport).Clone()
+	t.MaxIdleConns = 100
+	t.MaxConnsPerHost = 100
```

Comment:
MaxConnsPerHost default is 0 (unlimited), why do we want to scope it down here?

Also note that MaxIdleConns default is already 100, is this just a defensive measure in case that changes?

Reply:
That's MaxIdleConnsPerHost, not MaxConnsPerHost